higlass_manage/redis/redis.conf

```
@@ -0,0 +1,4 @@
+maxmemory 8gb
```

```suggestion
maxmemory 4gb
```
Safer bet for the majority of users :-)